deps/http_parser/http_parser.c

```diff
@@ -1572,7 +1572,7 @@ size_t http_parser_execute (http_parser *parser,
             REEXECUTE();
           }
 
-          if (!lenient && !IS_HEADER_CHAR(ch)) {
+          if (!lenient && !IS_HEADER_CHAR((unsigned char)ch)) {
```

Comment:
It's possible. I likely just missed that particular comment. If so, my apologies. Unfortunately, it also appears we didn't have any tests looking for this.